config.js

```diff
@@ -47,6 +47,34 @@ const config = {
           .map(s => (s ? Number(s) : null))
           .filter(i => i && !isNaN(i)),
   },
+  resell: {
+    // ms after limit order placed before it is able to be canceled due to not filling
+    cancel: multiply(process.env.CPBB_RESELL_CANCEL || 0, 60000),
+    pumps: process.env.CPBB_RESELL
+      ? process.env.CPBB_RESELL.split(',')
+          .map(p => {
+            if (!p) return;
+            return divide(p.replace('%', '').split('@')[1], 100);
+          })
+          .filter(i => i && !isNaN(i))
+      : (process.env.CPBB_RESELL_AT || '') // legacy support
```

Comment:
Let's remove the legacy support for this one. Maybe for Rebuy too since we are changing the REBUY_ONLY env var. We'll make this a breaking change major release.